joomla/joomla-cms
# [4.2] Add autocomplete=one-time-code attribute to mfa input field

### Summary of Changes
Added the autocomplete attribute to the mfa input field on the captive page. By default, "one-time-code" is used as the attribute value, but plugins can override it if necessary - see the "fixed" plugin, which set's the attribute to "off" (even though that setting will be ignored by many modern browsers, so consider this as a polite suggestion for the browser...).

As discussed with Nic, I'm doing this as a public PR as MFA codes are one-time-codes and therefore even a previously stored MFA token isn't a security issue.

### Testing Instructions
Setup a 4.2 site, configure MFA for an example user, verify the existence of the attribute on the MFA input field on the captive pager after login.


### Actual result BEFORE applying this Pull Request
No attribute.


### Expected result AFTER applying this Pull Request
Attribute.
